conjur-api/User.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="User.cs" company="Conjur Inc.">
+//     Copyright (c) 2016-2018 Conjur Inc. All rights reserved.
+// </copyright>
+// <summary>
+//     User manipulation routines.
+// </summary>
+
+namespace Conjur
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Conjur User reference.
+    /// </summary>
+    /// A user represents an identity for a human. It is a role, in RBAC terms.
```

Comment:
Nitpick: strictly speaking it's both a role and a resource, and here you're referencing the resource part.

Reply:
Change description to describe user as resource only